azure-pipelines.yml

```diff
@@ -47,7 +47,7 @@ stages:
       displayName: 'Execute Markdownlint'
       condition: eq(variables['build.reason'], 'PullRequest')
 
-    - script: build.cmd -ci
+    - script: build.cmd -ci /maxCpuCount:1
```

Comment:
can you elaborate on this? cc: @joperezr

Reply:
This is for testing only. It allows to see the output of all test runs in CI. I'll remove it when everything is ready.